source/manual/request-fastly-tls-certificate.html.md

```diff
@@ -0,0 +1,55 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Request Fastly TLS certificate
+section: Transition
+layout: manual_layout
+parent: "/manual.html"
+related_applications: [bouncer, transition]
+---
+
+Use the 2nd line account to login to Fastly.
+
+Go to Configure > Switch services
+
+Select “Production Bouncer” and search for the domain
+
+![Photo of the step 1](images/fastly/1.png)
```

Comment:
And all of these images could be cropped to remove the browser URL, the footer, the empty whitespace to the sides of the content, etc. It will make it easier to scan, and reduce the size of the images from their current ~0.5MB each.